package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eventemitter2",
-  "version": "2.1.3",
+  "version": "2.1.4",
```

Comment:
One comment, please leave the bumping of the version out of the PR so that we can do a version update when I merge a bunch of features at once